Gemfile

```diff
@@ -140,6 +140,7 @@ gem 'locale', '~> 2.0.0', '< 2.1.0'
 gem 'routing-filter', '~> 0.5.0', '< 0.6.0'
 gem 'unicode', '~> 0.4.0'
 gem 'unidecoder', '~> 1.1.0'
+gem 'money', '~> 6.10.0'
```

Comment:
nice but worth noting what it's for in the commit comment - it's obvious now but nice to have a note of the decision for far future people (or me in 3 weeks' time!)